rules/repositories.bzl

```diff
@@ -141,13 +143,6 @@ def _rules_ios_bzlmod_dependencies():
         ],
     )
 
-def _rules_ios_bzlmod_dev_dependencies():
```

Comment:
The git diff here seems wrong, the real change was moving `rules_pkg` from `_rules_ios_bzlmod_dev_dependencies` to `_rules_ios_bzlmod_dependencies`